runtime/oti/bytecodewalk.h

```diff
@@ -97,11 +97,11 @@ base types: (in the 19bit class index field)
 
 #define BCV_KNOWN_CLASSES							7
 
-#define BCV_OBJECT_OR_ARRAY					0		/* added for completeness */
-#define BCV_TAG_BASE_TYPE_OR_TOP			1		/* clear bit means object or array */
-#define BCV_TAG_BASE_ARRAY_OR_NULL			2		/* set bit means base type array */
-#define BCV_SPECIAL_INIT					4		/* set bit means special init object ("this" for <init>) */
-#define BCV_SPECIAL_NEW						8		/* set bit means special new object (PC offset in upper 28 bits) */
+#define BCV_OBJECT_OR_ARRAY					0
```

Comment:
Okay. I was thinking it would be good to have one set of comments, but I don't think there's a risk of this code changing significantly :) I'll add them back.